app/Repositories/Activity/ActivityRepository.php

```diff
@@ -355,6 +357,50 @@ public function populateOrderNumber()
         }
     }
 
+    /**
+     * Gets activities for LTI external tool search from CurrikiGo
+     * @param array $data
+     * @return array
+     */
+    public function ltiSearchForm ($request) {
```

Comment:
Formatting issue: 
no space after function name, and opening bracket should be in the new line.